src/ray/protobuf/ray_syncer.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
+option cc_enable_arenas = true;
+package ray.rpc.syncer;
+
+enum RayComponentId {
+  RESOURCE_MANAGER = 0;
+  SCHEDULER = 1;
+}
+
+message RaySyncMessage {
+  // The version of the message.
+  uint64 version = 1;
+  // The component this message is for.
+  RayComponentId component_id = 2;
+  // The actual payload.
+  bytes sync_message = 3;
```

Comment:
will this incur one more string copy?

Reply:
We have one extra copy to avoid one extra serialization when sending data.

Furthermore, I think in the future, the component actually can use flatbuffer to optimize for performance if needed.